databrowser/README.md

```diff
@@ -59,6 +59,12 @@ If you are in the root folder of this mono repository, change to the Databrowser
 cd databrowser
 ```
 
+Provide a `.env` file with necessary configurations. You can use the `.env.example` file as basis:
```

Comment:
Here I would adjust a bit the readme standard from the boilerplate: https://github.com/noi-techpark/java-boilerplate#execute-with-docker

Reply:
Do you mean that we should change the sentence to the one below (from the linked resource)?

`Copy the file .env.example to .env and adjust the configuration parameters.`

Reply:
Yes, I just know that they try to allign all readme files as much as possible. But this is just a minor point.

Reply:
I think that's a good argument. I changed the sentence to the one shown above.